.pre-commit-config.yaml

```
@@ -37,7 +37,7 @@ repos:
       # - id: fix-encoding-pragma  # covered by pyupgrade
       - id: trailing-whitespace
         # Trims trailing whitespace.
-        exclude: ".*(data.*|extern.*|licenses.*|_parsetab.py|test_cds.py)$"
+        exclude: ".*(data.*|extern.*|licenses.*|astropy.*.py)$"
```

I had a look at the `pre-commit` documentation and found that it is possible to [exclude files by type without having to write a regex](https://pre-commit.com/#filtering-files-with-types).
```suggestion
        exclude_types: [python]
        exclude: ".*(data.*|extern.*|licenses.*)$"
```